docs/types_of_spectra.rst

```diff
@@ -42,7 +44,11 @@ with their corresponding ``specutils`` representations:
    not rectangular), this case does not have a specific representation in
    ``specutils``.  Instead, this case should be dealt with by making lists (or
    numpy object-arrays) of `~specutils.Spectrum1D` objects, and iterating over
-   them.
+   them. Specutils does provide a `SpectrumList` class which is
```

Comment:
(minor)
After seeing this in the actual docs page I found the paragraph was getting a bit awkwardly long, so maybe just a stand-alone paragraph is better?  (Although my comment on the line below might lead you to changing this enough to negate this comment.)
```suggestion
   them. 
   
   Specutils does provide a `SpectrumList` class which is
```